cookbook/helper/scrapers/scrapers.py

```diff
@@ -15,7 +16,13 @@
 
 
 def text_scraper(text, url=None):
-    scraper_class = SchemaScraperFactory.SchemaScraper
+    domain = None
```

Comment:
@gloriousDan I'll have to look at how to refactor to use the scrape_html but work isn't going to give me the time to do this now.  
This PR restores the ability to use custom scraper (for sites behind paywall), I'll add a feature request to refactor.

Reply:
I think that with this implementation some scrapers are broken. Can you test with e.g. http://mobile.kptncook.com/recipe/pinterest/Low-Carb-Tarte-Flamb%C3%A9e-with-Serrano-Ham-%26-Cream-Cheese/315c3c32?lang=en if all fields especially the instructions are imported?

Reply:
OK - that's a bug.  The importer is getting raw HTML text then skipping the scraper because it already has the text and attempting to run html_scraper.  This breaks kptncook because that scraper is doing an api connection behind the scenes.

I hacked a fix in place - but that whole section needs to be refactored to take into account the changes at the front end.

Reply:
i dont remember exactly why but when i last checked there was an issue with the scrape_html function that caused me not to implement it yet. But dont remember what exactly was holding me of.

Reply:
@vabene1111 - it doesn't work when the URL is not provided.